yarn-project/ethereum/src/l1_tx_utils.ts

```diff
@@ -365,6 +370,15 @@ export class L1TxUtils {
     const block = await this.publicClient.getBlock({ blockTag: 'latest' });
     const baseFee = block.baseFeePerGas ?? 0n;
 
+    // Get blob base fee if available
+    let blobBaseFee = 0n;
+    try {
+      const blobBaseFeeHex = await this.publicClient.request({ method: 'eth_blobBaseFee' });
+      blobBaseFee = BigInt(blobBaseFeeHex);
```

Comment:
I understand we should be increasing the blob base fee to 12.5% per stall time in blocks, as we do with the base fee (see [here](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-4844.md#base-fee-per-blob-gas-update-rule)).